test/general/dpl_namespace.pass.cpp

```diff
@@ -24,13 +24,12 @@
 #include <tuple>
 
 #if TEST_DPCPP_BACKEND_PRESENT
-#   include <CL/sycl.hpp>
```

Comment:
Let's include ```"oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h"``` instead of ```sycl.hpp``` here and in the other tests (.cpp files in ```test``` folder) to avoid code duplications.

Reply:
Done